packages/protocol/test/test_integration.sh

```diff
@@ -29,7 +29,7 @@ docker run -d \
   --http.api debug,eth,net,web3,txpool,miner
 
 # Wait till the test node fully started
-RETRIES=30
+RETRIES=200
```

Comment:
Not sure whether this change is necessary, if retrying 30 times doesn't work, neither will retrying 200 times?

Reply:
will revert
```suggestion
RETRIES=30
```